saskatoon/harvest/filters.py

```diff
@@ -36,32 +38,41 @@ class HarvestFilter(filters.FilterSet):
             is_staff=True
         ),
         required=False,
+        lookup_expr="exact",
         help_text="",
     )
 
     trees = filters.ModelChoiceFilter(
         queryset=TreeType.objects.all(),
         label=_("Tree"),
+        lookup_expr="exact",
         help_text="",
         required=False
     )
 
     property__neighborhood = filters.ModelChoiceFilter(
         queryset=Neighborhood.objects.all(),
         label=_("Neighborhood"),
+        lookup_expr="exact",
         help_text="",
         required=False
     )
 
     class Meta:
         model = Harvest
-        fields = {
-        'status': ['exact'],
-        'pick_leader': ['exact'],
-        'trees': ['exact'],
-        'property__neighborhood': ['exact'],
-        'start_date': ['exact'],
-        }
+        fields = (
+            'status',
+            'pick_leader',
+            'trees',
+            'property__neighborhood',
+            'season',
+        )
+
+    def filter_season(self, queryset, name, value):
+        if value:
+            lookup = '__'.join([name, 'year'])
+            return queryset.filter(**{lookup: value})
+        return queryset
```

Comment:
is this working? name will be `season` but you want to look for `start_date__year`

Reply:
Yes, I have tested it and it's working. The line `lookup = '__'.join([name, 'year'])` returns `start_date__year`